lib/datasource/metadata.ts

```diff
@@ -132,7 +133,9 @@ export function addMetaData(
       dep[url] = dep[url].trim();
       // istanbul ignore if
       if (!dep[url].match(/^https?:\/\//)) {
-        delete dep[url];
+        // logger.debug({ url, dep}, '(not) Deleting non https')
+        // TODO: I commented this out but I'm not sure we want to keep it this way
+        // delete dep[url];
```

Comment:
Why? Do you need non https changelog urls?

Reply:
fixed